README.md

```diff
@@ -58,52 +58,30 @@ npm install @google-cloud/compute
 ### Using the client library
 
 ```javascript
+/**
+ * TODO(developer): Uncomment these variables before running the sample.
+ */
+// const projectId = 'YOUR_PROJECT_ID';
+// const zone = 'europe-central2-b'
+
 const compute = require('@google-cloud/compute');
 
-function sleep(timeoutMs) {
-  return new Promise(resolve => {
-    setTimeout(resolve, timeoutMs);
+async function listInstances() {
+  const client = new compute.InstancesClient({fallback: 'rest'});
```

Comment:
What happens if the customer doesn't set `fallback: 'rest'`?  Or is that going to be required for every instance instantiation?

Reply:
In this case user will use grpc transport. GAPIC team will fix it (rest will be by default)